### ISSUE TITLE
Middleware throws instead of rejecting if request is invalid

### ISSUE THREAD

**Issue by Vendicated** [discord/discord-interactions-js#13]
when sending an invalid request without the proper headers, instead of rejecting, the middleware throws an error:
```js
                throw new Error('Value is not a valid hex string');
                       ^
Error: Value is not a valid hex string
     at valueToUint8Array (/home/ven/Bots/tldr-discord/node_modules/discord-interactions/dist/index.js:89:23)
```

I sent this request via
```bash
curl -XPOST https://domain/interactions
```

This is my first time using express, so I'm not sure if this is the intended behaviour. But I think it would be better to simply reject if the request is invalid

Edit: After inspecting the code, it seems like this is already being handled correctly, so it confuses me that I am getting this error: Any clue what could be causing this?

**Comment by typpo** [comment]
Can you check what version of this package you have installed?

**Comment by Vendicated** [comment]
Just checked the local code actually and the offending block is not in the try catch, so I seem to not have the latest code. I just installed it without targetting a specific version

**Comment by typpo** [comment]
Ok, try specifying 2.0.0 and see how that works!

**Comment by Vendicated** [comment]
```bash
$ npm ls discord-interactions
tldr-discord@1.0.0 /home/ven/Bots/tldr-discord
└── discord-interactions@2.0.0```

The installed version already is at 2.0.0 and even installing via @latest doesnt solve it

**Comment by typpo** [comment]
I just pushed 2.0.1 which should correctly bring in changes that were on `main`.  Can you see how that works?

**Comment by Vendicated** [comment]
I already turned my pc off so I had to do it via termux

 It's still using the old code for some reason

![Screenshot_20210304-062503__01](https://user-images.githubusercontent.com/45497981/109915631-6ffff380-7cb2-11eb-982e-72cbdc9c43b8.jpg)

It is really late here so I'll be going to bed now. Thanks a lot for the quick response and have a great rest of your day/night :) I'll respond tomorrow if you need anything else from me

**Comment by typpo** [comment]
Let's give it one more shot with 2.0.2

**Comment by Vendicated** [comment]
I'm sorry, I forgot to reply. Yes it's working wonderfully now! Although I think the console.error should either be made optional or be turned into a callback you can pass into the middleware 

Thanks again for the very quick replies and this great module!
